ios/Flutter/AppFrameworkInfo.plist

```diff
@@ -7,7 +7,7 @@
 	<key>CFBundleExecutable</key>
 	<string>App</string>
 	<key>CFBundleIdentifier</key>
-	<string>com.joinseeds.seedslight</string>
+  	<string>io.flutter.flutter.app</string>
```

Comment:
Sure?

Reply:
This was the bug - extremely important to leave this like that. That's what cost me hours.

Reply:
this is the bundle ID for an _internal framework_
If it is set to the same ID as the app id, it won't install on the phone.

Reply:
Very good to know